src/vivarium/framework/population/interface.py

```diff
@@ -126,12 +132,12 @@ def initializes_simulants(
             required_resources,
         )
 
-    def register_source_columns(self, component: Component | Manager) -> None:
-        """Registers the source columns created by a component or manager.
+    def register_private_columns(self, component: Component | Manager) -> None:
```

Comment:
sorry, couldn't help it. I'm actually leaning more towards referring to them as "source" columns, but that would be a bigger change and just wanted to get consistent for now since we refer to them as "private" throughout